src/tes/logger/log.go

```diff
@@ -22,6 +23,26 @@ type Logger interface {
 	WithFields(...interface{}) Logger
 }
 
+// SetLogLevel sets the level of logging
+func SetLogLevel(l string) {
+	switch strings.ToLower(l) {
+	case "debug":
+		logrus.SetLevel(logrus.DebugLevel)
+	case "info":
+		logrus.SetLevel(logrus.InfoLevel)
+	case "warn":
+		logrus.SetLevel(logrus.WarnLevel)
+	case "error":
+		logrus.SetLevel(logrus.ErrorLevel)
+	case "fatal":
+		logrus.SetLevel(logrus.FatalLevel)
+	case "panic":
+		logrus.SetLevel(logrus.PanicLevel)
+	default:
+		logrus.SetLevel(logrus.DebugLevel)
```

Comment:
In linux commands you usually opt-in to debug logging, so maybe a better default is info